packages/crayons-extended/custom-objects/src/index.html

```diff
@@ -452,1076 +452,1432 @@
-            updated_at: '2023-03-14 14:32:38',
-          },
-          {
-            id: '19680ca2-69b2-4c59-a56a-4188a15190ea',
-            name: 'cf_sdsdds',
-            column_name: 'cf_boolean001',
-            label: 'sdsdds',
-            type: 5,
-            field_options: {
-              unique: 'false',
-              required_on_resolve: 'false',
-            },
-            custom: true,
-            editable: true,
-            position: 85,
-            deleted: false,
-            required: false,
-            visible: true,
-            column_type_blob: false,
-            nodegroup_skip: false,
-            validatable: false,
-            builder: false,
-            internal: false,
-            choices: [],
-            fields: [],
-            regex: {},
-            created_at: '2023-03-14 14:32:57',
-            updated_at: '2023-03-14 14:32:57',
-          },
-          {
-            id: '4d804b91-9963-4a30-b5b0-957c52552141',
-            name: 'cf_checkboaas',
-            column_name: 'cf_boolean004',
-            label: 'checkboaas',
-            type: 5,
-            field_options: {
-              unique: 'false',
-              required_on_resolve: 'false',
-            },
-            custom: true,
-            editable: true,
-            position: 86,
-            deleted: false,
-            required: false,
-            visible: true,
-            column_type_blob: false,
-            nodegroup_skip: false,
-            validatable: false,
-            builder: false,
-            internal: false,
-            choices: [],
-            fields: [],
-            regex: {},
-            created_at: '2023-03-14 14:33:15',
-            updated_at: '2023-03-14 14:33:15',
-          },
-          {
-            id: '56689136-1c8e-4915-9f8e-f423b7de4991',
-            name: 'cf_heyhasas',
-            column_name: 'cf_boolean005',
-            label: 'heyhasas',
-            type: 5,
-            field_options: {
-              unique: 'false',
-              required_on_resolve: 'false',
-            },
-            custom: true,
-            editable: true,
-            position: 87,
-            deleted: false,
-            required: false,
-            visible: true,
-            column_type_blob: false,
-            nodegroup_skip: false,
-            validatable: false,
-            builder: false,
-            internal: false,
-            choices: [],
-            fields: [],
-            regex: {},
-            created_at: '2023-03-14 14:33:21',
-            updated_at: '2023-03-14 14:33:21',
-          },
-          {
-            id: '8937a82b-e632-477d-8a7d-17e0d7c8cf17',
-            name: 'cf_heye',
-            column_name: 'cf_boolean003',
-            label: 'heye',
-            type: 5,
-            field_options: {
-              unique: 'false',
-              required_on_resolve: 'false',
+            "id": "888c6dad-30b2-4030-b070-a704428609fe",
```

Comment:
Can we not commit huge changes in index.html - This file is just for local testing. We can avoid commits in this file